src/pages/program/index.js

```diff
@@ -24,7 +25,12 @@ const index = () => {
   return (
     <>
       <div>
-        <Navbar />
+        <Navbar gaLabel="p"/>
```

Comment:
In this case both program/index and program/local were not using the layout component and I didn't change it thinking there would be a reason they were 'different' from the other pages :D For consistency, should I change them so that they have all the content wrapped in a `<Layout>` ?

Reply:
Yes maybe that was once a dump thing from me :-D Would be nice to only have one :-)